### PR TITLE
[WISE-289] removed printStackTrace stmts and related code cleanup

### PR DESCRIPTION
**Pull request by rsearls**

### CODE REVIEW

File: gui/src/main/java/org/jboss/wise/gui/ResponseLogHandler.java [hunk 1]
```diff
@@ -35,14 +35,16 @@
 import javax.xml.ws.handler.LogicalHandler;
 import javax.xml.ws.handler.LogicalMessageContext;
 import javax.xml.ws.handler.MessageContext;
+import org.jboss.logging.Logger;
+import org.jboss.wise.shared.WsdlFinder;
```
Comment (asoldano) [inline gui/src/main/java/org/jboss/wise/gui/ResponseLogHandler.java]: Is the WsdlFinder import really needed?

File: gui/src/main/java/org/jboss/wise/gui/ResponseLogHandler.java [hunk 2]
```diff
@@ -83,7 +85,8 @@ private void doLog(LogicalMessageContext smc) {
 	    } catch (Exception e) {
 		PrintWriter ps = new PrintWriter(outputStream);
 		ps.println("Exception getting response message log: ");
-		e.printStackTrace(ps);
+		//debug only  e.printStackTrace(ps);
+			log.error(e);
```
Comment (asoldano) [inline gui/src/main/java/org/jboss/wise/gui/ResponseLogHandler.java]: The e.printStackTrace(ps) here was actually printing the stacktrace on the provided outputStream; by printing it to the logger you're actually breaking this functionality.

File: gui/src/main/java/org/jboss/wise/shared/GWTClientConversationBean.java [hunk 3]
```diff
@@ -140,7 +140,8 @@ public RequestResponse parseOperationParameters(String curOperation) {
          setCurrentOperationFullName(currentOperationFullName);
 
       } catch (Exception e) {
-         e.printStackTrace();
+         //e.printStackTrace();
```
Comment (asoldano) [inline gui/src/main/java/org/jboss/wise/shared/GWTClientConversationBean.java]: You can actually remove the commented line at this point.

File: gui/src/main/java/org/jboss/wise/shared/WsdlFinder.java [hunk 4]
```diff
@@ -42,6 +43,7 @@
 public class WsdlFinder implements Serializable {
 
    private static final long serialVersionUID = -6841057226758482815L;
+   private static Logger log = Logger.getLogger(WsdlFinder.class);
```
Comment (asoldano) [inline gui/src/main/java/org/jboss/wise/shared/WsdlFinder.java]: why isn't this 'final'?

File: gui/src/main/java/org/jboss/wise/shared/WsdlFinder.java [hunk 5]
```diff
@@ -58,7 +60,8 @@ public WsdlFinder() {
             }
          }
       } catch (Exception e) {
-         e.printStackTrace();
+         //e.printStackTrace();
```
Comment (asoldano) [inline gui/src/main/java/org/jboss/wise/shared/WsdlFinder.java]: you can remove the line instead of commenting it

File: gui/src/main/java/org/jboss/wise/gui/ResponseLogHandler.java [hunk 6]
```diff
@@ -35,14 +35,15 @@
...
+import org.jboss.logging.Logger;
 
 /**
  * Logs the client inbound response message
  * 
  * @author alessio.soldano@jboss.com
  */
 public class ResponseLogHandler implements LogicalHandler<LogicalMessageContext> {
-
+	private static Logger log = Logger.getLogger(ResponseLogHandler.class);
```
Comment (asoldano) [inline gui/src/main/java/org/jboss/wise/gui/ResponseLogHandler.java]: This is not needed (nothing is using it), please remote it.

File: gui/src/main/java/org/jboss/wise/gwt/client/view/EndpointConfigView.java [hunk 7]
```diff
@@ -87,7 +87,6 @@
    public EndpointConfigView() {
 
       SimplePanel contentDetailsDecorator = new SimplePanel();
-      contentDetailsDecorator.setWidth("100%");
```
Comment (asoldano) [inline gui/src/main/java/org/jboss/wise/gwt/client/view/EndpointConfigView.java]: Just as a confirmation, is this on purpose? Asking as the change does not look related to WISE-289

**Comment by rsearls** [comment]
Removed unused Logging stmt.
